designs/odcr.md

```diff
@@ -0,0 +1,213 @@
+      #    - open:
+      #       The Capacity Reservation accepts all instances that have
+      #       matching attributes (instance type, platform, and Availability
+      #       Zone). Instances that have matching attributes launch into the
+      #       Capacity Reservation automatically without specifying any
+      #       additional parameters.
+      #    - targeted:
+      #       The Capacity Reservation only accepts instances that
+      #       have matching attributes (instance type, platform, and
+      #       Availability Zone), and explicitly target the Capacity
+      #       Reservation. This ensures that only permitted instances can use
+      #             the reserved capacity.
+      type: String | None
+status:
+  capacityReservations:
+    - # AvailabilityZone of the Capacity Reservation
+      availabilityZone: String
+      # Available Instance Count of the Capacity Reservation
+      availableInstanceCount: Integer
+      # The date and time at which the Capacity Reservation expires. When a Capacity
+      # Reservation expires, the reserved capacity is released and you can no longer
+      # launch instances into it. The Capacity Reservation's state changes to expired
+      # when it reaches its end date and time.
+      endDate: String | None
+      # Indicates the way in which the Capacity Reservation ends. A Capacity Reservation
+      # can have one of the following end types:
+      #   * unlimited - The Capacity Reservation remains active until you explicitly
+      #   cancel it.
+      #   * limited - The Capacity Reservation expires automatically at a specified
+      #   date and time.
+      endDateType: String
+      # ID of the Capacity Reservation
+      id: String
+      # Indicates the type of instance launches that the Capacity Reservation accepts. The options include:
+      #   - open:
+      #       The Capacity Reservation accepts all instances that have
+      #       matching attributes (instance type, platform, and Availability
+      #       Zone). Instances that have matching attributes launch into the
+      #       Capacity Reservation automatically without specifying any
+      #       additional parameters.
+      #   - targeted:
+      #       The Capacity Reservation only accepts instances that
+      #       have matching attributes (instance type, platform, and
+      #       Availability Zone), and explicitly target the Capacity
+      #       Reservation. This ensures that only permitted instances can use
+      #       the reserved capacity.
+      instanceMatchCriteria: String
+      # Instance Platform of the Capacity Reservation
+      instancePlatform: String
+      # Instance Type of the Capacity Reservation
+      instanceType: String
+      # Owner Id of the Capacity Reservation
+      ownerId: String
+      # The date and time at which the Capacity Reservation was started.
+      startDate: String
```

Comment:
How relevant do we think that this is? I think we should only surface active reservations through this status so I'm not sure how useful knowing the startDate is going to be